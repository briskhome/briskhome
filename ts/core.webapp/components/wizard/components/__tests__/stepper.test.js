import React from 'react';
import toJson from 'enzyme-to-json';
import { render } from 'enzyme';
import { WizardStepper } from "../../";
describe('core.webapp -> components -> wizard -> stepper', () => {
  it('renders', () => {
    expect(toJson(render(<WizardStepper currentSlide={0} totalSlides={1} />))).toMatchSnapshot();
  });
});