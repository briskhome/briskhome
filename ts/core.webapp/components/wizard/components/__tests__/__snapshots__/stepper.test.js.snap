// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core.webapp -> components -> wizard -> stepper renders 1`] = `
<ul
  class="briskhome-wizard__stepper"
>
  <li
    class="briskhome-wizard__step briskhome-wizard__step_active"
  >
    <span
      class="briskhome-wizard__step-circle"
    >
      1
    </span>
  </li>
</ul>
`;
